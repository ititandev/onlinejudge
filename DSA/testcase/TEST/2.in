10
73.76 -37.31 -25.34 -66.59 -33.7 -91.35 -10.48 -61.95 -66.35 -40.28
