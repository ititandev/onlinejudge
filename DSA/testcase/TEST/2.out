  -91.35  -66.59  -66.35  -61.95  -40.28
  -37.31   -33.7  -25.34  -10.48   73.76
